sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/query.sql

```diff
@@ -262,7 +262,7 @@ _current AS (
     baseline_ping AS baseline
   USING
     (client_id)
-  LEFT JOIN
+  FULL OUTER JOIN
```

Comment:
Looks good, we could validate that numbers don't change after a backfill or add a test --It shouldn't be the case since activations are based on baseline ping.

Reply:
It looks like there are activations that do not exist in baseline ping.  I added an inner join to the existing table.  Let me know what you think.